### PR TITLE
OM-66: move mPass redirect to the utils

### PR DESCRIPTION
**Pull request by jdolkowski**
https://openimis.atlassian.net/browse/OM-66

### CODE REVIEW

File: src/helpers/utils.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import { logout } from "../actions";
+import {baseApiUrl, logout} from "../actions";
+import {M_PASS_LOGOUT_PATH} from "../constants";
```
Comment (olewandowski1) [inline src/helpers/utils.js:2]: Add extra space here

**Review by olewandowski1 — CHANGES_REQUESTED** [review]
@jdolkowski I left 1 comment.

File: src/helpers/utils.js (hunk 1 above)
Comment (jdolkowski) [reply src/helpers/utils.js]: done

**Review by olewandowski1 — APPROVED** [review]
LGTM @jdolkowski

**Pull request merged**